src/components/accessories/patientNewBill/NewBillSide.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import SmallButton from "../smallButton/SmallButton";
+import TextButton from "../textButton/TextButton";
+import "./styles.scss";
+
+interface IOwnProps {
+  handlePaymentDialog: (value: boolean) => void;
+}
+
+const NewBillSide: FC<IOwnProps> = ({ handlePaymentDialog }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="two-row">
+        <span>Total:</span>
+        <span>120$</span>
+      </div>
+      <div className="two-row">
+        <span>Paid:</span>
+        <span>0$</span>
+      </div>
+      <div className="xs-divider"></div>
+      <div className="two-row">
+        <span>To be paid:</span>
+        <span>120$</span>
+      </div>
+      <div>
+        <SmallButton
+          onClick={() => {
+            handlePaymentDialog(true);
+          }}
+        >
+          {t("bill.pay")}
+        </SmallButton>
+      </div>
+      <div className="lg-divider"></div>
+      <div>
+        <TextButton onClick={() => {}}>{t("button.suspend")}</TextButton>
+      </div>
+      <div>
+        <TextButton onClick={() => {}}>{t("button.delete")}</TextButton>
+      </div>
```

Comment:
Just for now, put a trash icon inside the delete button and a stop / wait icon inside the suspend button.

Reply:
And I think that's fine to add the Refund button in order to undo a payment. 
Will be my task discuss with Ilario about his implementation and API.

Reply:
For the refund button, we keep it for the moment as you will have to discuss with Ilario